docs/releasing.md

```diff
@@ -1,8 +1,36 @@
-## Release guide for maintainers
+### Release guide for maintainers
+
+## Release Policy
+
+### Cadence
+
+We ship twice a month, on the Thursday before the 15th and the Thursday before 1st. These dates are added to the internal Lighthouse calendar.
+
+### Release manager
+
+Rather than a rotation, release manager is appointed. However if the appointed manager is absent, the next engineer in the following list owns it:
+
+    bckenny, paulirish, patrickhulce
+
+Release manager follows the below _Release Process_.
+
+### Release publicity
+
+1. Release mgr copies changelog to a new [Releases](https://github.com/GoogleChrome/lighthouse/releases). Tags and ships it.
```

Comment:
Also- when we writeup the release log- can we make sure to mention when the release will land in DT?

Reply:
good idea. we can say "We expect this release to ship in the DevTools of Chrome XX" though we can't necessarily promise.

Reply:
Yup yup- just to give folks an idea. If we can add that in to the doc, that would be great.